### ISSUE TITLE
Unable to access Craftbeerpi v2.2 via ios/macos

### ISSUE THREAD

**Issue by mactore88** [Manuel83/craftbeerpi#126]
I am having issues accessing Craftbeerpi v2.2 from my ios devices and my macbook. No problem acessing it via my android smartphone or PC. I have just done a clean install of the new software, but still no luck in accessing it. Does anyone have a solution for this?

**Comment by duzvik** [comment]
nope, today i've made a clean install and everything is OK. Except temperature  graphs :(
Check that mac is connected to same wi-fi network.

**Comment by DrLaban** [comment]
Did you figure out the graphs @duzvik ? Delete the tempsensor logfiles - then the graphs should work again.

**Comment by duzvik** [comment]
@DrLaban Thanks. This helps. 
Same here:
https://github.com/Manuel83/craftbeerpi/issues/113
